cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/BatchFileFilter.java

```diff
@@ -85,7 +90,7 @@ public boolean accept(Path path) {
       LOG.warn("Couldn't parse file: " + path.getName());
       fileDate = prevMinute;
```

Comment:
I'm unsure, but if there's a file with a "malformed" name, this'll mean that we read it every run, right?
Instead, would it be better to not read it or also propagate the exception?